EasyPost.Tests/ModelsTests/CarrierDetailTest.cs

```diff
@@ -0,0 +1,11 @@
+using EasyPost.Tests._Utilities;
```

Comment:
Question: What's the purpose of these files if there are no tests in them?

Reply:
Leftover dead code, will remove.

Reply:
Ok, there are a couple dozen so be sure to catch them all.

Reply:
Resolved